packages/google_sign_in/google_sign_in_web/android/src/main/java/io/flutter/plugins/google_sign_in_web/GoogleSignInWebPlugin.java

```diff
@@ -0,0 +1,44 @@
+package io.flutter.plugins.google_sign_in_web;
+
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+
+/** GoogleSignInWebPlugin */
+public class GoogleSignInWebPlugin implements FlutterPlugin, MethodCallHandler {
```

Comment:
nit: this doesn't need to extend MethodCallHandler (delete onMethodCall)

Reply:
Done.